src/worker/read-wordlist.ts

```diff
@@ -38,6 +38,33 @@ export async function readExcel(
   return wordlist;
 }
 
+export async function readTSV(TSVFile: File): Promise<WordList> {
+  return new Promise((resolve, reject) => {
+    const wordlist: WordList = [];
+
+    const reader = new FileReader();
+    reader.addEventListener("load", (event: ProgressEvent<FileReader>) => {
+      if (event.target && typeof event.target.result === "string") {
+        // get file contents
+        const rows = event.target.result.split("\n");
+        for (let i = 1; i < rows.length; i++) {
+          const row = rows[i].split("\t");
+          if (row.length !== 2) {
```

Comment:
In `readExcel()` we will still read a line even if it has more than two columns. Could you compare the two functions to ensure that the two functions behave similarly? 

Big bonus points if you find a way to extract that kind of logic out as a common function